ledger/ledger-api-auth/src/test/suite/scala/com/digitalasset/ledger/api/auth/AuthServiceJWTCodecSpec.scala

```diff
@@ -58,12 +56,60 @@ class AuthServiceJWTCodecSpec
     )
 
   // participantId is mandatory for the format `StandardJWTTokenFormat.ParticipantId`
-  private val StandardJWTPayloadGen = Gen.resultOf(StandardJWTPayload).filterNot { payload =>
-    !payload.participantId
-      .exists(_.nonEmpty) && payload.format == StandardJWTTokenFormat.ParticipantId
+  private val StandardJWTPayloadGen = Gen
+    .resultOf(StandardJWTPayload)
+    .filterNot { payload =>
+      !payload.participantId
+        .exists(_.nonEmpty) && payload.format == StandardJWTTokenFormat.ParticipantId
+    }
+    // we do not fill audiences for Scope or ParticipantId based tokens
```

Comment:
why not? you can still set an audience there no?

Reply:
actually I dont need this change, removing

Reply:
actually I need this. For the tests defined with this generator, "aud" is not written in the jwt writer, so this field is not really isomorphic